pages/Authentication/logout/index.html

```diff
@@ -217,5 +222,7 @@ <h3 class="text-uppercase auth-head flex-row flex-align-center flex-justify-cent
                 </div>
             </div>           
         </footer>
+        <script src="/scripts/hamburger.js"></script>
+        <script src="/scripts/hamburger.js"></script>
```

Comment:
Typo

Reply:
Thank you for pointing this out! I shall update this after the review is done. 👍🏽